docs/contributors/code/native-mobile-integration-test-guide.md

```diff
@@ -338,7 +332,7 @@ fireEvent.press( buttonInnerBlock );
 
 If you have trouble locating an element’s identifier, you may wish to use Xcode’s Accessibility Inspector. Most identifiers are cross-platform, so even though the tests are run on Android by default, the Accessibility Inspector can be used to find the right identifier.
 
-<img src="../../assets/xcode-accessibility-inspector-screenshot.png" alt="Screenshot of the Xcode Accessibility Inspector app. The screenshot shows how to choose the correct target in the device dropdown, enable target mode, and locate accessibility labels after tapping on screen elements"/>
+<img src="https://raw.githubusercontent.com/WordPress/gutenberg/trunk/docs/assets/xcode-accessibility-inspector-screenshot.png" alt="Screenshot of the Xcode Accessibility Inspector app. The screenshot shows how to choose the correct target in the device dropdown, enable target mode, and locate accessibility labels after tapping on screen elements"/>
```

Comment:
Embedding images is not covered explicitly in the [documentation docs](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/documentation/README.md), so I'm using this format which is used in other places such as [here](https://github.com/WordPress/gutenberg/blame/trunk/docs/contributors/code/release.md#L31).